src/features/marketplace/store/marketplace.actions.ts

```diff
@@ -93,6 +135,51 @@ export const buyItem = createAsyncThunk(
   }
 );
 
+export const changePriceItem = createAsyncThunk(
+  'marketplace/changePriceItem',
+  async (
+    {
+      ownerAddress,
+      price,
+      itemId,
+      listId,
+    }: {
+      ownerAddress: string;
+      price: string;
+      listId: number;
+      itemId: string;
+    },
+    { dispatch }
+  ) => {
+    try {
+      const marketplaceState = {
+        ownerAddress,
+        collection: CollectionType.LedaNft,
+        collectionAddress: LedaAddress.address,
+        mintEventName: ContractEvent.LogChangePrice,
+        price,
+        itemId,
+        listId,
+        status: ItemStatus.Listed,
+      } as MarketplaceState;
+
+      const processor = new MarketplaceClientProcessor();
+      const listed = await processor.execute(marketplaceState);
+
+      dispatch(
+        openToastSuccess('The item has been successfully changed price on the marketplace.')
```

Comment:
Change the label to something like 'The item price has been successfully changed"